javaharness/java/arcs/android/AndroidPecPortManager.java

```diff
@@ -0,0 +1,84 @@
+package arcs.android;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import arcs.api.ParticleExecutionContext;
+import arcs.api.PecPort;
+import arcs.api.PecPortManager;
+import arcs.api.PecPortProxy;
+import arcs.api.PortableJson;
+import arcs.api.PortableJsonParser;
+import arcs.api.ShellApi;
+
+public class AndroidPecPortManager implements PecPortManager {
```

Comment:
why is this class in `arcs.andorid` package? it seems to be generic, could be in arcs.api, no?

Reply:
Merged back.